test/test-source-map-consumer.js

```diff
@@ -1119,3 +1119,16 @@ exports['test consuming names and sources that are numbers'] = function (assert)
   });
   assert.equal(i, 1);
 };
+
+exports['test from issue 227'] = function (assert) {
```

Comment:
I would appreciate a little more verbosity on the test name. `test non-normalized sourceRoot values (from issue #227)`.